webui/install/bareos/bareos-dir.d/profile/webui-admin.conf

```diff
@@ -3,7 +3,7 @@
 #
 Profile {
   Name = "webui-admin"
-  CommandACL = !.bvfs_clear_cache, !.exit, !.sql, !configure, !create, !delete, !purge, !prune, !sqlquery, !umount, !unmount, *all*
+  CommandACL = !.bvfs_clear_cache, !.exit, !.sql, !create, !delete, !purge, !prune, !sqlquery, !umount, !unmount, *all*
```

Comment:
What for the `configure` command is needed? With `configure` allowed, all other ACL restrictions are useless.

Reply:
Maybe @arogge can give us some insides why this is the case, I did not expect the behavior as well.

Reply:
Well, the case is obvious. For `status subscription` you need to know all resources. Internally, a check for the configure command ACL is often (mis-)used to detect, if you have "unlimited" access.
I'm not sure, what resources `status subscription` really check. I guess, this check could get replaced by checks for access to ALL clients and ALL jobs. Maybe some more. So I guess, this is what is required. Next question is, who will implement this and should this get implemented inside this PR or a separate one.

Reply:
... I guess, `status subscription` will also work with limited access on jobs and clients. However, than information will than not cover the full system.

Reply:
For the record the error message you would receive.
`
string(208) "{"jsonrpc":"2.0","id":null,"error":{"code":1,"message":"failed","data":{"result":{},"messages":{"error":["status subscription: is an invalid command or needs access right to the \"configure\" command.\n"]}}}}"
`

Reply:
https://github.com/bareos/bareos/pull/1280/commits/4b627ffc2e6beb9877563da722eb85197e9a7445 should eliminate our concerns giving access to the configure command, as it would not be required anymore.

Reply:
Rebased once again, https://github.com/bareos/bareos/commit/441032cf7ea1355fa35724d80ad16b28863a1e19 should be a better solution.

Reply:
outdated